### PR TITLE
docs: gpload - new yaml file parameter STAGING_TABLE

### PR DESCRIPTION
**Pull request by mkiyama**
Ported from 4.3.x documentation.

PR for 5X_STABLE
Will be ported to MAIN

 see PR https://github.com/greenplum-db/gpdb/pull/4356

### CODE REVIEW

File: gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml [hunk 1]
```diff
@@ -733,6 +734,34 @@
                                 <pd>Optional. If set to true, <codeph>gpload</codeph> will remove
                                     all rows in the target table prior to loading it.</pd>
                             </plentry>
+                            <plentry>
+                                <pt id="cfstagetbl">STAGING_TABLE</pt>
+                                <pd>Optional. Specify a table name to use as the temporary external
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml]: Suggest rewording to:  "Specify a temporary table name for gpfdist to use during a gpload operation."

File: gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml [hunk 2]
```diff
@@ -733,6 +734,34 @@
...
+                                        <varname>external_table_name</varname>. If the
+                                        <codeph>SCHEMA</codeph> value is <codeph>%</codeph>,
+                                        <varname>external_table_name</varname> is created in same
+                                    schema as the <xref href="#topic1/cftable" format="dita"
+                                        >TABLE</xref> in the <codeph>OUTPUT</codeph> section (the
+                                    target table being loaded into). </pd>
+                                <pd>If <codeph>SCHEMA</codeph> is not set,
+                                        <varname>external_table_name</varname> is created in the
+                                        <codeph>PUBLIC</codeph> schema. </pd>
+                                <pd>If the <varname>external_table_name</varname> exists, the
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml]: Should probably also clarify that if the table doesn't exist it's created with the same table schema as the OUTPUT table?

File: gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml [hunk 3]
```diff
@@ -733,6 +734,34 @@
...
+                                    utility uses the existing table. The utility returns an error if
+                                    the table schema is not the same as the a
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml]: Typo:  "as the a"

File: gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml [hunk 4]
```diff
@@ -733,6 +734,34 @@
...
+                                        <codeph>gpload</codeph> operation. <codeph>REUSE_TABLES:
+                                        true</codeph> must also specified. If
+                                        <codeph>REUSE_TABLES</codeph> is false or not specified,
+                                        <codeph>STAGING_TABLE</codeph> is ignored.</pd>
+                                <pd>If <varname>external_table_name</varname> contains a period (.),
+                                        <codeph>gpload</codeph> returns an error.</pd>
+                                <pd><codeph>gpload</codeph> uses the value of <codeph><xref
+                                            href="#topic1/cfschema" format="dita"
+                                        >SCHEMA</xref></codeph> in the <codeph>EXTERNAL</codeph>
+                                    section as the schema for the
```
Comment (lisakowen) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml]: external_table_name is the name of a table.  i don't think we want to preface external_table_name with "the".  i.e. if the name of the table was sales, would this sound correct "... as the schema for the sales." ?

File: gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml (hunk 1 above)
Comment (mkiyama) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml]: I avoided "temporary table" because that is a type of "normal" table. 
I thought it was misleading/confusing to use that term with external tables.

Comment (dyozie) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml]: Makes sense.  So should it be "temporary external table" as you had it originally?  Or just "Specify an external table name for gpfdist to use..."?

Comment (dyozie) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpload.xml]: Maybe you could add some explanation as to when the table is removed.

**Comment by mkiyama** [comment]
pushed commit with updates based on review and other comments.

**Comment by mkiyama** [comment]
@lij55 @huiliang-liu 
Have you had a chance to review the doc updates for STAGING_TABLE?
